src/array/__tests__/arrayMax-text.js

```diff
@@ -0,0 +1,9 @@
+import arrayMax from '../arrayMax';
+import { expect } from 'chai';
+
+describe('arrayMax', () => {
+    it('should return the maximum value from the given array list', () => {
```

Comment:
What happens when there are no items?
Does it work correctly when there are two items with the same value?